cap-containers/map.h

```diff
@@ -191,25 +328,21 @@ static int cap_map_insert(cap_map *map, void *key, void *value) {
 		}
 	}
 	cap_map *new_node = (cap_map *)CAP_ALLOCATOR(cap_map, 1);
-	if (!new_node) {
```

Comment:
This modification should be reverted.

Please take the changes from the current `map.h` file since there are some bug fixes.